coursebuilder/modules/explorer/_static/components/pages/homepage.html

```diff
@@ -186,15 +186,15 @@ <h2 class="section-label">Testimonials</h2>
       <template is="dom-if" if="[[!currentUser.loggedIn]]">
         <a href="[[currentUser.loginUrl]]">
           <div class="continue-learning">
-            <a class="waves-effect waves-light btn-large" target="_blank">Start Learning</a>
+            <a class="waves-effect waves-light btn-large" target="_blank" href="[[currentUser.loginUrl]]">Start Learning</a>
```

Comment:
@Sharonsyra let's remove target="_blank" for this.

Reply:
will do

Reply:
done